set.mm

```diff
@@ -43525,6 +43525,34 @@ same distinct variable group (meaning ` A ` cannot depend on ` x ` ) and
   $}
 
   ${
+    $d x A $.  $d x B $.
+    iuneq1d.1 $e |- ( ph -> A = B ) $.
```

Comment:
There is a lot to be said for keeping the antecedent in exactly the same format (cf. `iuneq1dv`) for automated proof generation by recursion.